.github/workflows/usermods.yml

```
@@ -0,0 +1,71 @@
+name: Usermod CI
+
+on:
+  push:
+    paths:
+      - usermods/**
+      - .github/workflows/usermods.yml
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)